elastic/kibana
# [ILM] Account for deprecated prop in show request flyout

Fixes https://github.com/elastic/kibana/issues/177604

## Summary

Account for deprecated prop in show request flyout, if present.

##### How to test

* Start up kibana and es
* Navigate to Stack management -> ILM
* Verify that:
  * Opening a deprecated policy and hitting the show request button shows the deprecated flag
  * Opening a non deprecated policy and hitting the show request button doesnt show the deprecated flag


![Screenshot 2024-03-22 at 09 55 27](https://github.com/elastic/kibana/assets/1191206/17870b85-ced8-403b-9ee0-a3d2e41051bf)
